### DISCUSSION TITLE
HELP Changes in plugins through AudioGridder are not being saved by Cubase

### DISCUSSION THREAD

**Discussion by Loupap** [apohl79/audiogridder#1312] [Help] [answered]
Any help would be appreciated on this.

I am using Cubase 11 on a fully speced out 2013 Mac Trashcan (12 Core - Max Ram - Etc..). I am running an AudioGridder Server on a 2nd dedicated fully maxed out 2013 Mac Pro Traschan. System runs without any glitches except for the following:

When I open any plugin through Audiogridder and make changes to the chosen plugin UI in the Audiogridder window......cubase will not save the changes I make unless I move my mouse to the top of the Audiogridder window, away from the plugin UI and then click my mouse.....ONLY THEN IT WILL ALLOW ME TO SAVE IN CUBASE.

Any suggestions please?

Thanks

**Comment by emack999** [comment, marked as answer]
You answered your own question :)
"...move my mouse to the top of the Audiogridder window, away from the plugin UI and then click my mouse"

**Reply by Loupap** [reply]
Why does it work that way? Any other plugin that is not going through Audio Gridder saves regardless of the mouse location.

Is that because the plugins are within a plugin??

**Reply by kcoul** [reply]
yes, I don't think the DAW can track the "Dirty State" of the plugin otherwise, therefore it doesn't think there are any changes available to save until this is done.
